Spine/Transformer.swift

```diff
@@ -156,6 +156,8 @@ It uses the date format configured in the DateAttribute.
 private struct DateTransformer: Transformer {
 	func formatter(attribute: DateAttribute) -> NSDateFormatter {
 		let formatter = NSDateFormatter()
+        let timeZone = NSTimeZone(name: "UTC")
+        formatter.timeZone = timeZone
```

Comment:
My indentation is correct, I don't know why this is like this :(

<img width="442" alt="screen shot 2015-07-17 at 8 01 17 pm" src="https://cloud.githubusercontent.com/assets/70312/8758778/a743b4a2-2cbe-11e5-9033-52333af9feb9.png">